Makefile

```diff
@@ -0,0 +1,4 @@
+logstash-installation:
+	curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o logstash/awscliv2.zip
+	curl https://artifacts.elastic.co/downloads/logstash/logstash-oss-7.16.3-linux-x86_64.tar.gz -o logstash/logstash-oss-7.16.3-linux-x86_64.tar.gz
```

Comment:
The logstash version is hard-coded in a few places, do we want to consolidate that variable somewhere?